### DISCUSSION TITLE
SD Maid 2

### DISCUSSION THREAD

**Discussion by Roger1465** [d4rken-org/sdmaid-se#798] [Q&A]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Smartphone (please complete the following information):**
 - Device: [e.g. Pixel 9]
 - OS: [e.g. Android 17]
 - Root status [e.g. Rooted - Magisk v25.2]
 - SD Maid 2/SE version [e.g. v0.6.6-rc6]

**Debug log**
If you can reproduce the bug, record the event in [a debug log](https://github.com/d4rken-org/sdmaid-se/wiki/Bugs#debug-log).

**Screenshots/Video**
If applicable, add screenshots or a video if it helps to explain your problem.

**Additional context**

Hallo,ist es Normal,das die App Bereinigung bei SD Maid 2 /SE, 12 Std. nach Auto.Bereinigung, Manuel Erneut 1,7GB Frei macht/bereinigt? Obgleich das Gerät gar nicht genutzt wurde! Das Erweckt in mir den Eindruck,das mein Zimmermädchen,am Abend zuvor nix Automatisch bereinigt hat!?!Obgleich es angezeigt wird.Es wäre sehr Freundlich,wenn mir jemand helfen kann!

**Comment by d4rken** [comment]
Caches die nur mit der Bedienungshilfe gelöscht werden können und diese funktioniert nicht im Hintergrund (d.h. via Planer).

 >The accessibility service can't run in the background so it won't execute when run via Scheduler.
> If you only clean user apps, and not system apps, then you can set up Shizuku to execute the same operation in the background via Scheduler.

**Reply by Roger1465** [reply]
Hallo,d4rken,danke für Deine Antwort!
Aber Shizuku habe ich wieder aufgegeben! Außerdem gefällt mir das mit der Entwickleroption nicht! Also habe ich  Dich so verstanden,daß SD Maid 2 meine Caches,meiner Benutzer-Apps garnicht Autom. über den Planer(Scheduler) bereinigt,ja!? Aber wofür erteile ich dann noch die Barrierefreiheit?
Sorry bin kein Computer-Nerd,von daher war mein Versuch Shizuku einzurichten schon eine Herausforderung:))

**Reply by d4rken** [reply]
> Also habe ich Dich so verstanden,daß SD Maid 2 meine Caches,meiner Benutzer-Apps garnicht Autom. über den Planer(Scheduler) bereinigt,ja!?

Das betrifft hauptsächlich die Standard Caches, an SD Maid anders nicht ran kommt. Hängt alles etwas davon ab welche Android Version und auch wo Apps ihre Daten abgelegt haben.

> Aber wofür erteile ich dann noch die Barrierefreiheit?

Für wenn du die gleiche Operation ohne Scheduler ausführst, während SD Maid offen ist.

**Comment by Roger1465** [comment]
Okay Matthias, ich sehe schon das wird mir zu Kompliziert,sorry! Habe meine
Qualitäten in anderen Bereichen! Also hoffe ich ,daß SD Maid 2 schon für
irgendwas jut ist! Zumindestens war es bei SD Maid 1 so! Und ich glaube für
meine wenigen Internetaktionen reicht das was mein Zimmermädchen
macht,völlig aus! Und Manuell mache ich,bis auf ein paar Ausschlüsse,so gut
wie nix!Keine Manuelle Bereinigung! Ich finde bei SD Maid 2 auch keine
Historie, wie bei 1 ? Du siehst lieber Matthias, alles was über die
Autom.Bereinigung hinausgeht,überfordert mich,sorry!
Also ich danke Dir für Deine Netten Bemühungen und wünsche Dir noch eine
gute Zeit!
Gruß Roger.

R.D

Matthias Urhahn ***@***.***> schrieb am So., 12. Nov. 2023,
10:39:

**Comment by MarkyMarkDE** [comment]
ja, Shizuku einzurichten ist etwas knifflig. Hinzu kommt der Faktor, dass man bei dem Neustart des Geräts das Ding wieder zb. per WLAN unlock Code aktivieren und paaren muss. Auf nicht gerooteten Geräten macht dieses Tool leider null Sinn, da dann halt kein Autostart von Shizuku möglich ist.
Ich hatte es auch eingerichtet und mich dann auch wieder getrennt, Entwicklermodus plus jedes Mal neu paaren ist es mir echt nicht Wert.

**Comment by Roger1465** [comment]
Hallo MarkyMark,
Danke für Deine Info! Aber kannst Du mir sagen,warum ich seit neuem die
Appbereinigung Manuell betätigen muß! Habe gestern Abend schonwieder die
Message bekommen,das ein Fehler aufgetreten ist und ich es Manuell starten
soll! Das war sonst nie und von SD Maid 1 kenne ich das nicht,da war immer
alles Tuti !
Für Deine Hilfreiche Antwort sage ich schonmal Dankeschön!
Gruß Roger.

MarkyMarkDE ***@***.***> schrieb am Fr., 8. Dez. 2023, 21:02:

> ja, Shizuku einzurichten ist etwas knifflig. Hinzu kommt der Faktor, dass
> man bei dem Neustart des Geräts das Ding wieder zb. per WLAN unlock Code
> aktivieren und paaren muss. Auf nicht gerooteten Geräten macht dieses Tool
> leider null Sinn, da dann halt kein Autostart von Shizuku möglich ist.
> Ich hatte es auch eingerichtet und mich dann auch wieder getrennt,
> Entwicklermodus plus jedes Mal neu paaren ist es mir echt nicht Wert.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/d4rken-org/sdmaid-se/discussions/798#discussioncomment-7803154>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BCZIY6JES7B3PO7BA5EBU4LYINW6VAVCNFSM6AAAAAA7HEZPSGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMBTGE2TI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
R.D
